autopush/web/health.py

```diff
@@ -58,12 +59,12 @@ def _check_error(self, failure, name):
         self.log.failure(format=fmt, failure=failure, name=name)
 
         cause = self._health_checks[name] = {"status": "NOT OK"}
-        if failure.check(InternalServerError):
+        if failure.check(InternalServerError):      # pragma nocover
             cause["error"] = "Server error"
-        elif failure.check(MissingTableException):
+        elif failure.check(MissingTableException):  # pragma nocover
             cause["error"] = failure.getErrorMessage()
         else:
-            cause["error"] = "Internal error"
+            cause["error"] = "Internal error"       # pragma nocover
```

Comment:
We had tests that covered these, now we don't?

Reply:
Unfortunately, the test that originally triggered that had to be dropped, and getting the code to exhibit the behavior was far more difficult than expected, mostly due to the `thread.local` element isolating portions of the code from each other. I should revisit this code and see if I can enforce this behavior, but for now, I felt it's ok to have this as a "catch" behavior for edge cases.

Reply:
Added #1114